types/networks/Cargo.toml

```diff
@@ -8,7 +8,6 @@ edition = "2021"
 anyhow             = "1.0"
 fil_actors_runtime = "=8.0.0"
 forest_beacon      = "0.1.0"
-forest_fil_types   = "0.2"
 forest_json        = "0.1.0"
 fvm_shared         = { version = "0.8.0", default-features = false }
 lazy_static        = "1.4"
```

Comment:
`lazy_static` is also unused but `cargo udeps` can't see that.

Reply:
Fixed!